common.groovy

```diff
@@ -23,7 +23,7 @@ defaults = [
     webhookURL: 'a53b3a9e-d649-4cff-9997-6c24f07743c8',
   ],
   helm: [
-    version: 'v2.3.1',
+    version: 'v2.5.0',
```

Comment:
might want to bump to v2.5.1 instead; there was a few pretty serious bugs in 2.5.0.

https://github.com/kubernetes/helm/releases/tag/v2.5.1